stdlib/3/os/__init__.pyi

```diff
@@ -213,7 +213,7 @@ if sys.version_info >= (3, 6):
         def is_dir(self, follow_symlinks: bool = ...) -> bool: ...
         def is_file(self, follow_symlinks: bool = ...) -> bool: ...
         def is_symlink(self) -> bool: ...
-        def stat(self) -> stat_result: ...
+        def stat(self, follow_symlinks: bool = ...) -> stat_result: ...
```

Comment:
Please mark this argument as a keyword-only argument.

```suggestion
        def stat(self, *, follow_symlinks: bool = ...) -> stat_result: ...
```

(And maybe do the same with `is_dir` and `is_file` while you are at it.)